extensions/pyRevitTools.extension/checks/radar_check.py

```diff
@@ -0,0 +1,624 @@
+# -*- coding: utf-8 -*-
+# region Imports
```

Comment:
I started using them on the large functions, thank you for the tip.
![image](https://github.com/user-attachments/assets/776d4fc7-e695-4036-be58-2e1012b220df)

Reply:
Note that if you find yourself using regions in functions because they are large, probably they are too large, and regions could become separate, shorter and simpler functions 😉